### PR TITLE
gpde: fix error handing parameters

### PR DESCRIPTION
**Pull request by BadAssassin**
Fixes usage of Rast3d_fatal_error()

### CODE REVIEW

File: lib/gpde/n_arrays_io.c [hunk 1]
```diff
@@ -363,7 +363,7 @@ N_array_3d *N_read_rast3d_to_array_3d(char *name, N_array_3d * array,
 
     /* Close files and exit */
     if (!Rast3d_close(map))
-        Rast3d_fatal_error(map, NULL, 0, _("Error closing g3d file"));
+        Rast3d_fatal_error(_("Error closing g3d file"));
```
Comment (wenzeslaus) [inline lib/gpde/n_arrays_io.c:366]: `name` seems to be available here (with the right content). Can you please change it as you did in #2597?

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
Thank you. If it is possible, please change it to show the name.

File: lib/gpde/n_arrays_io.c (hunk 1 above)
Comment (nilason) [reply lib/gpde/n_arrays_io.c]: I took the liberty to add this.

**Review by wenzeslaus — APPROVED** [review]
Looks good. Thanks @BadAssassin and @nilason!

**Pull request merged**